sqlrunner/main.py

```diff
@@ -34,6 +34,10 @@ def main(args):
     write_results(results, args["output"])
 
 
+def generate_column_headers(dummy_data_file, sql_query):
```

Comment:
I think we should call this function `get_column_headers`, as I don't think we will implement it as a [generator](https://docs.python.org/3.10/glossary.html#term-generator). I also think this function should accept the query and return the column headers in a single-element iterable, meaning that it can be passed to the `write_results` function.

```py
def get_column_headers(sql_query):
    raise NotImplementedError
```